docker/skopeo_runner/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM fedora:33
```

Comment:
Is this the image you intend to use in production?  Can you share some info on how it'll be built?  It is best to have some kind of plan how to ensure this will be regularly rebuilt, and updated when F33 isn't supported any more.

In another repo I've had success with approach:

- use a specific digest in FROM
- activate renovatebot
- it will automatically bump the digest when a new version is available, e.g. https://github.com/rohanpm/ssh-prometheus-exporter/pull/9
- github updates trigger quay rebuild at https://quay.io/repository/rmcgover/ssh-prometheus-exporter

That way the repo keeps track of a known good image, and there's some automated prompt to get it regularly rebuilt.

Reply:
you could also use github action to start building the image & pushing it to quay whenever there is a change in your dockerfile. here is an example: https://github.com/zanssa/iib-deployment/blob/main/.github/workflows/quay.yml

Reply:
I added renovatebot settings and Github action for building it (just as a test). I'll add a Quay trigger after this PR is merged.